apollo-router/src/apollo_studio_interop/mod.rs

```diff
@@ -391,21 +467,41 @@ fn format_selection_set(selection_set: &SelectionSet, f: &mut fmt::Formatter) ->
     Ok(())
 }
 
-fn format_variable(arg: &Node<VariableDefinition>, f: &mut fmt::Formatter) -> fmt::Result {
+fn format_variable(
+    arg: &Node<VariableDefinition>,
+    normalization_algorithm: &SignatureNormalizationAlgorithm,
+    f: &mut fmt::Formatter,
+) -> fmt::Result {
     write!(f, "${}:{}", arg.name, arg.ty)?;
     if let Some(value) = &arg.default_value {
         f.write_str("=")?;
-        format_value(value, f)?;
+        format_value(value, normalization_algorithm, f)?;
     }
-    format_directives(&arg.directives, false, f)
+
+    // The JS implementation doesn't sort directives (this is overridden in enhanced mode)
+    format_directives(&arg.directives, false, normalization_algorithm, f)
 }
 
-fn format_argument(arg: &Node<Argument>, f: &mut fmt::Formatter) -> fmt::Result {
+fn format_argument(
+    arg: &Node<Argument>,
+    normalization_algorithm: &SignatureNormalizationAlgorithm,
+    f: &mut fmt::Formatter,
+) -> fmt::Result {
     write!(f, "{}:", arg.name)?;
-    format_value(&arg.value, f)
+    format_value(&arg.value, normalization_algorithm, f)
 }
 
-fn format_field(field: &Node<Field>, f: &mut fmt::Formatter) -> fmt::Result {
+fn format_field(
+    field: &Node<Field>,
+    normalization_algorithm: &SignatureNormalizationAlgorithm,
+    f: &mut fmt::Formatter,
+) -> fmt::Result {
+    if is_enhanced(normalization_algorithm) {
+        if let Some(alias) = &field.alias {
+            write!(f, "{}:", alias)?;
```

Comment:
```suggestion
            write!(f, "{alias}:")?;
```
Not a big deal but subjectively looks nicer